src/app/core/ui/navigation/menu-item.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, Input } from "@angular/core";
+import { CommonModule } from "@angular/common";
+import { MatListModule } from "@angular/material/list";
+import { FaIconComponent } from "@fortawesome/angular-fontawesome";
+import { FaDynamicIconComponent } from "../../common-components/fa-dynamic-icon/fa-dynamic-icon.component";
+import { RouterLink } from "@angular/router";
+import { Angulartics2Module } from "angulartics2";
+import { NgForOf } from "@angular/common";
+import { MenuItem } from "./menu-item";
+import { MatMenuModule } from "@angular/material/menu";
+
+@Component({
+  selector: "app-menu-item",
+  templateUrl: "./menu-item.component.html",
+  styleUrls: ["./menu-item.component.scss"],
+  imports: [
+    CommonModule,
+    MatListModule,
+    FaIconComponent,
+    FaDynamicIconComponent,
+    RouterLink,
+    Angulartics2Module,
+    NgForOf,
+    MatMenuModule,
+  ],
+  standalone: true,
+})
+export class MenuItemComponent {
+  @Input() item: MenuItem; // Receives each individual menu item as input
+  activeLink: string;
```

Comment:
what is "activeLink" used for?

Reply:
To keep track of the currently active route or link within the menu.

Reply:
Yes, right  I should pass the activeLink value as an @Input to MenuItemComponent.

Reply:
> It is working for me now, Let me know that whether it is working well for you also.

Nope, not working for me @sadaf895 . Did you run the app and see the menu item highlighted correctly? We need to get back to a design where the currently visited menu is white background (see screenshots)

In the code I don't see you passing in any value to the `activeMenuItem` input also.